scripts/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+# Script to bootstrap repo for development
+
+set -e
+
+# install dependencies
+yarn
+
+ROOT_MODULES=`pwd`/node_modules
```

Comment:
Nit: Rename to `ROOT_NODE_MODULES_DIR` to distinguish from other `modules` being handled in this file.